mnemonic-core/src/main/java/org/apache/mnemonic/VolatileMemAllocator.java

```diff
@@ -29,11 +29,12 @@
  * 
```

Comment:
Please delete .DS_Store from your commit and add it to .gitignore for avoiding track this file.
Please squash all your commits/changes to one commit.